### ISSUE TITLE
variables missing from drop-down

### ISSUE THREAD

**Issue by jpritikin** [stan-dev/shinystan#189]
There are at least two bugs,

1. Line 104 of the R script fails. Apparently, some code expects the model to be passed as the variable name `fit`
2. Once shinystan is launched, I can browse the vector parameters `twin`, `family`, `rawTwin`, `rawFamily` but none of the non-vector parameters `predVar`, `eVar`, etc

See [test.tar.gz](https://github.com/stan-dev/shinystan/files/5988243/test.tar.gz)

**Comment by jpritikin** [comment]
Bug 1 above seems fixed in the `v3-alpha` branch. Bug 2 seems to affect both branches.

**Comment by jpritikin** [comment]
Oops, you can comment `tall[['ord']] <- ordinalize(tall[['value']], 12)` and `source("util.R")` to get the script to run.

**Comment by VeenDuco** [comment]
Hey, happy to hear you got it to work in the end.

**Comment by jpritikin** [comment]
> Hey, happy to hear you got it to work in the end.

No, my comment was in reference to the test script that I attached to the initial report. The bug(s) in shinystan are not fixed.

**Comment by VeenDuco** [comment]
Okay, good to know. I've reopened the issue. Not quite sure when I'll be able to take a deep dive but will keep you posted. Kind regards,

Duco
